corescore.core

```diff
@@ -103,6 +103,13 @@ filesets:
       - data/vivado_waive.tcl : {file_type : tclSource}
       - data/vcu118.xdc : {file_type : xdc}
 
+  haps_dx7:
+    files:
+      - rtl/corescore_haps_dx7_clock_gen.v : {file_type : verilogSource}
+      - rtl/corescore_haps_dx7.v : {file_type : verilogSource}
+      - data/vivado_waive.tcl : {file_type : tclSource}
```

Comment:
Did you forget to add this file? If so, just amend the commit, or otherwise drop from the file list

Reply:
This vivado_waive file was already in the data dir for a different Xilinx target. I did not need any changes since it seemed related to intervals. Is that the file you mean?

Reply:
Haha. Thought I had done a thorough review when I found this. Completely missed that the file already existed. Sorry about that. All good. Picked and pushed. Many thanks for your contribution